components/ProfileBoardBlock.tsx

```diff
@@ -0,0 +1,9 @@
+export default function Block(guess: Number) {
```

Comment:
`Number` should be `number` (type instead of class)

Reply:
Also there is a jpeg in the public folder that is never referenced (was probably a placeholder that is no longer necessary)